src/MOLFiniteDifference/MOL_discretization.jl

```diff
@@ -5,14 +5,15 @@ struct MOLFiniteDifference{T,T2} <: DiffEqBase.AbstractDiscretization
     time::T2
     upwind_order::Int
     centered_order::Int
+    grid_align::String
```

Comment:
We should use enums here. `@enum GridAlign EdgeAlign CenterAlign`. That would naturally catch any syntax issues.